src/main/java/org/apache/pulsar/manager/service/impl/BookiesServiceImpl.java

```diff
@@ -33,26 +35,35 @@
 @Service
 public class BookiesServiceImpl implements BookiesService {
 
+    private final EnvironmentCacheService environmentCacheService;
+    private final HttpServletRequest request;
     @Value("${backend.directRequestBroker}")
     private boolean directRequestBroker;
 
     @Value("${backend.directRequestHost}")
     private String directRequestHost;
 
-    @Value("${bookie.host}")
-    private String bookieHost;
+//    @Value("${bookie.host}")
```

Comment:
Please remove commented code

Reply:
@eolivelli 
The commented code has been deleted

Reply:
thanks